### PR TITLE
reg-compare: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1908

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/reg-compare.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Compare a specific value under two keys:
+
+`reg compare {{first_key_name}} {{second_key_name}} /v {{value}}`
+
+- Compare all sub keys and values for two keys:
+
+`reg compare {{first_key_name}} {{second_key_name}} /s`
+
+- Only output the differences between the specified keys (default):
```
Comment (sbrl) [inline pages/windows/reg-compare.md]: Is this example needed if it's the default operation mode, or could we remove it / merge it into the previous example?

Comment (sbrl) [inline pages/windows/reg-compare.md]: Is this example needed if it's the default operation mode, or could we remove it / merge it into the previous example?

Comment (pxgamer) [reply pages/windows/reg-compare.md]: I guess it can be removed. Just thought it was worth mentioning.

**Comment by sbrl** [comment]
Thanks again, @pxgamer :D

**Pull request merged**